Apps/Admin/Tests/Functional/cypress/integration/ui/dashboard/dashboard.cy.js

```diff
@@ -0,0 +1,92 @@
+const { localDevUri } = require("../../../support/constants");
+import { verifyTestingEnvironment } from "../../../support/functions/environment";
+
+describe("Dashboard", () => {
+    beforeEach(() => {
+        verifyTestingEnvironment();
+
+        cy.intercept("GET", "**/Dashboard/RegisteredCount*", (req) => {
+            req.reply({
+                fixture: "DashboardService/registered-count.json",
+            });
+        });
+
+        cy.intercept("GET", "**/Dashboard/LoggedInCount*", (req) => {
+            req.reply({
+                fixture: "DashboardService/logged-in-count.json",
+            });
+        });
+
+        cy.intercept("GET", "**/Dashboard/DependentCount*", (req) => {
+            req.reply({
+                fixture: "DashboardService/dependent-count.json",
+            });
+        });
+
+        // used to calculate [data-testid=average-rating]
+        cy.intercept("GET", "**/Dashboard/Ratings/Summary*", (req) => {
+            req.reply({
+                fixture: "DashboardService/summary.json",
+            });
+        });
+
+        // matches [data-testid=total-unique-users]
+        cy.intercept("GET", "**/Dashboard/RecurringUsers?days=3*", (req) => {
+            req.reply({
+                body: 2,
+            });
+        });
+
+        cy.log("Logging in.");
+        cy.login(Cypress.env("idir_username"), Cypress.env("idir_password"));
+    });
+
+    afterEach(() => {
+        cy.log("Logging out.");
+        cy.logout();
+    });
```

Comment:
This pattern seems to go against the suggested [best practices](https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks).  Might be worth discussing with team.

Reply:
I will remove in next PR.